From effe281f3017ad4aa09d6de8af86df41d848c2ed Mon Sep 17 00:00:00 2001
From: Vishal Sagar <vishal.sagar@xilinx.com>
Date: Mon, 4 May 2020 22:08:10 -0700
Subject: [PATCH 1351/1852] staging: xlnxsync: Fix the uapi header license

commit 0b197959bbbdb68e1da974bd013339f08704b178 from
https://github.com/Xilinx/linux-xlnx.git

Error: missing "WITH Linux-syscall-note" for SPDX-License-Identifier

Reported-by: kbuild test robot <lkp@intel.com>
Signed-off-by: Vishal Sagar <vishal.sagar@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 include/uapi/linux/xlnxsync.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/uapi/linux/xlnxsync.h b/include/uapi/linux/xlnxsync.h
index 9b3c89fbcd33..12f10a8e5a84 100644
--- a/include/uapi/linux/xlnxsync.h
+++ b/include/uapi/linux/xlnxsync.h
@@ -1,4 +1,4 @@
-/* SPDX-License-Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 WITH Linux-syscall-note */
 
 #ifndef __XLNXSYNC_H__
 #define __XLNXSYNC_H__
-- 
2.31.1

